### DISCUSSION TITLE
Goodbye Microsoft ???

### DISCUSSION THREAD

**Discussion by FakelsHub** [sfall-team/sfall#427] [Q&A] [answered]
It seems that the day has come when I have to say goodbye to GitHub
The reason is the refusal of MS support for some browsers.
That is, as before a few years ago, when I and others were struggling with this constant Microsoft problem in support of different browsers.
But now the problem has become worse, in addition, the old Opera browsers (last version on Android 4.2.2) stopped working, where I was constantly browsing and writing comments. Now all this will not happen. Perhaps I will find a new browser acceptable to me to view your activity and code.
But I am very demanding of the Internet browser, for example, Google browser and the like are considered very big freaks for me, which I will never use.

Maybe Microsoft will fix something in its policy in this problem with browsers.

While I'm waiting for the Github problem to be fixed.
You should not expect any answers/comments from me yet... (commits will be sent but in silence mode).

**Comment by egornovivan** [comment, marked as answer]
Из альтернатив только жирнолис, в котором гуй перекручивают каждый год

**Reply by FakelsHub** [reply]
Firefox или подобное попробую на Андроиде. но на таком старом андроиде скорее всего тоже не будет web интерфейс работать.
Помню давно что Firefox тоже перестала работать с гитхаб. поэтому я пересел на ее клон PaleeMoon.
А так ты прав Firefox подпортилась.

Сейчас для меня это так на моем основном браузере. зависание, и ничего нельзя сделать.
![Screenshot](https://user-images.githubusercontent.com/21695363/153256798-d4a5a4f3-2d90-48df-9dac-1fc76c3b75bd.jpg)

Десктопная Опера еще работает что радует пока.

**Reply by FakelsHub** [reply]
Я больше переживаю за андроид, что не смогу через него ничего сделать. я там больше активничал чем у компа.
Буду надеяться что Web заработает. Но такое в первый раз, чтобы и на андроиде браузер не работал с в web гита.

**Reply by egornovivan** [reply]
Попробуй Firefox esr, он не такой глючный как основная версия.

**Comment by Lexx2k** [comment]
I don't understand, what's the issue? I'm using the website fine with Firefox on Windows.

**Comment by NovaRain** [comment]
I've been using Firefox (newer versions) on Windows. On my Android phone (ASUS Zenfone Live L2) I'm using Brave browser. Sadly nowadays when it comes to browsers, it's either Chrome/Chromium based or Firefox.
As for Pale Moon, it seems related to JavaScript support: https://forum.palemoon.org/viewtopic.php?f=70&t=27879

I suppose you can still push commits via git.exe?

**Reply by FakelsHub** [reply]
> Chrome/Chromium based or Firefox.
> As for Pale Moon, it seems related to JavaScript support:

The thing is that for me these are very bad browsers for Windows. Using them is like pissing against the wind)))
Not only Palemon refused to work and the old Opera too.

**Reply by FakelsHub** [reply]
Sometimes these graphic innovations are already annoying. I want stability from the platform, and not to see every time changing buttons or possible unnecessary animations.

**Reply by NovaRain** [reply]
> The thing is that for me these are very bad browsers for Windows. Using them is like pissing against the wind)))
Not only Palemon refused to work and the old Opera too.

If by "bad" you mean memory/resource hog, I'm afraid there are not many choices now.
The old Gecko-based Opera is about obsolete, and Pale Moon might take a long time catching up the support.

> Sometimes these graphic innovations are already annoying. I want stability from the platform, and not to see every time changing buttons or possible unnecessary animations.

If it's just for the look, there are a couple of ways to keep Firefox closer to the old style. At least that's what I do for my Firefox.

**Reply by FakelsHub** [reply]
Wow. The GitHub Web is working again. Microsoft has fixed its bug.

**Comment by fedprod** [comment]
Ah yea, the problems with intermediate Browservariations that are no longer correctly supported in one case or another...
you could always just use Git Desktop or the CLI Tools, you know ^^"

**Comment by FakelsHub** [comment]
At the moment GitHub does not work even in Firefox 78 ESR

**Fucking M$!**
![153508596-f05c2aa6-9585-4505-aa94-b6a66143bc5d](https://user-images.githubusercontent.com/21695363/153529033-9ebcb158-7833-45aa-8963-795d1849b98a.png)
.

**Reply by NovaRain** [reply]
Looks like the blog post got removed from the changelog.

**Reply by FakelsHub** [reply]
But microsoft is up to something very bad.

**Reply by FakelsHub** [reply]
```
GitHub Support wrote:

    We have deployed a change today that should improve functionality on older and
    unsupported browsers.
```

**Comment by FakelsHub** [comment]
But all the old browsers don't work my all day again.
